sdk/file_sdk/Common/ifile_format.h

```diff
@@ -0,0 +1,40 @@
+#ifndef IFILE_FORMAT_H
+#define IFILE_FORMAT_H
+
+#include <string>
+#include <vector>

```

Comment:
there is no group for includes...
check other classes, there is no new line

---
In reply to: [134421002](https://github.com/AzureAD/rms-sdk-for-cpp/pull/133#discussion_r134421002) [](ancestors = 134421002)